code/index.html

```diff
@@ -4,27 +4,104 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@300&display=swap" rel="stylesheet">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
```

Comment:
Generellt snygg layout! Pluspoäng till din contact sida! Formuläret och accordion radar upp sig fint i större storlekar. Du har en fin effekt på bilden som zoomar in när du kommer upp i storlek!  🤩